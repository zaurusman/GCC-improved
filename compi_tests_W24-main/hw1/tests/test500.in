()"\0 45982 {"28921 t \0 \r \t \xC6 \xeA $ \" \n \" ckBWG2 G 6ef 36016 PfW M5nXO Q  C5Q0gv}!=!=
=="\\ I2F9Aybh<=voidcontinuewhile"\t\t \\\ \" "blmIFygP F t \x68 i I2F9Aybh"\x21 \0w i22603 d \r B K \\ \0 \x52 qaeRmuJmuw \K dcXPdh\x2F 69610 \\ "\n 98675 \n bool"m V23157 \" 072551 \n41076 81916 65372gm\" \xA2 87256 41400 F int"22839 cqLzUq rC \r QTHjsP "76756 78955cl3 udZ4s47Zo;gMSM8j38HlEY
tlROiV405c2b"T KvL7ztecvN5M \r >="wnfTeP 58506 97036 \r"S \t Gc2IDOyM5tBh Q \\\ \r\\\ if("wnfTeP 58506 97036 \rhqN
bool"\r b \x4E \t 88086 66896 lI88fC \r \x70 \r 21847 {!=>="79866 \0 \" b 69329 61372 \\ apzq t  "\r b \x4E \t 88086 66896 lI88fC \r \x70 \r 21847 "79866 \0 \" b 69329 61372 \\ apzq t "A sowCVHk \\ F \n\n 82517 10939 n \n Ibw \0 \r returnbAE58true/true"\t \" 66083 \x5A "\x66 28572 6886 \x92 n T \t \t \t GnlJD 98552 ASQ # \\ V continue/continue"4738 \" \W ksvBn t\p nED9 \x75 25595 \x0B \r C \\ \\\ \r uJs njfV \"F0rHc "4738 \" \W ksvBn t\p nED9 \x75 25595 \x0B \r C \\ \\\ \r uJs njfV \"F0rHc or==odm9qEF-continue( "\r XOPLJI6uxzJ \0 \0 56225 \" \0 m 001949 k5GU2f 19297 \0 \0 \r \\ \H \n 
qv3aU0t6aYj09cwDugaN5gRIifqv3aU0t6aYj0return"48755 t \x3 i \x23 \t void"37012 \" \n \0 \\ \" \xC5\0 x \" c 27873 \" \0 \t \xA3 \\ intnot
Pz-voidcontinue<"d\t cpZ0snT33dmiTu Y y 95708 h , \n 77180 87213 "\x3C 25176 56804 \\ 99218 \n \0 "\x3C 25176 56804 \\ 99218 \n \0 continue
byte"u \n \\ \" \n 40050 L1ZYjhC1Mz2yL \n S \0 o 643766472 \P\0 \\ e\r"nmq\" YmO1o F13605 \t\t uT2nDys \r 40668 \r 32913 \xEA 59658 l continueCo48LvdRUCUcGpint{lJG0kOkITg1S5eK/ 
