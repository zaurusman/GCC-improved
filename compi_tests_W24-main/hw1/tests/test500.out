1 LPAREN (
1 RPAREN )
1 STRING 
1 NUM 28921
1 ID t
Error \
